extensions/ql-vscode/src/vscode-tests/no-workspace/data/remote-queries/queries/MRVA Integration test 1-6sBi6oaky_fxqXW2NA4bx/11111111/results.sarif

```diff
@@ -28,7 +28,7 @@
**\n   * CLI version where the `--allow-library-packs` option to `codeql resolve queries` was\n   * introduced.\n   */\n  public static CLI_VERSION_WITH_ALLOW_LIBRARY_PACKS_IN_RESOLVE_QUERIES = new SemVer('2.6.1');\n\n  /**\n   * CLI version where the `database unbundle` subcommand was introduced.\n   */\n  public static CLI_VERSION_WITH_DATABASE_UNBUNDLE = new SemVer('2.6.0');\n\n  /**\n   * CLI version where the `--no-precompile` option for pack creation was introduced.\n   */\n  public static CLI_VERSION_WITH_NO_PRECOMPILE = new SemVer('2.7.1');\n\n  /**\n   * CLI version where remote queries are supported.\n   */\n  public static CLI_VERSION_REMOTE_QUERIES = new SemVer('2.6.3');\n\n  /**\n   * CLI version where the `resolve ml-models` subcommand was introduced.\n   */\n  public static CLI_VERSION_WITH_RESOLVE_ML_MODELS = new SemVer('2.7.3');\n\n  /**\n   * CLI version where the `--old-eval-stats` option to the query server was introduced.\n   */\n  public static CLI_VERSION_WITH_OLD_EVAL_STATS = new SemVer('2.7.4');\n\n  /**\n   * CLI version where packaging was introduced.\n   */\n  public static CLI_VERSION_WITH_PACKAGING = new SemVer('2.6.0');\n\n  constructor(private readonly cli: CodeQLCliServer) {\n    /**/\n  }\n\n  private async isVersionAtLeast(v: SemVer) {\n    return (await this.cli.getVersion()).compare(v) >= 0;\n  }\n\n  public async supportsDecompileDil() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_DECOMPILE_KIND_DIL);\n  }\n\n  public async supportsLanguageName() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_LANGUAGE);\n  }\n\n  public async supportsDowngrades() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_DOWNGRADES);\n  }\n\n  public async supportsResolveQlref() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_RESOLVE_QLREF);\n  }\n\n  public async supportsAllowLibraryPacksInResolveQueries() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_ALLOW_LIBRARY_PACKS_IN_RESOLVE_QUERIES);\n  }\n\n  async supportsDatabaseRegistration() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_DB_REGISTRATION);\n  }\n\n  async supportsDatabaseUnbundle() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_DATABASE_UNBUNDLE);\n  }\n\n  async supportsNoPrecompile() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_NO_PRECOMPILE);\n  }\n\n  async supportsRemoteQueries() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_REMOTE_QUERIES);\n  }\n\n  async supportsResolveMlModels() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_RESOLVE_ML_MODELS);\n  }\n\n  async supportsOldEvalStats() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_OLD_EVAL_STATS);\n  }\n\n  async supportsPackaging() {\n    return this.isVersionAtLeast(CliVersionConstraint.CLI_VERSION_WITH_PACKAGING);\n  }\n}\n"
```

Comment:
Let's keep this file unchanged since it's test data and the code in it will never be executed.